command/module.go

```diff
@@ -77,18 +79,20 @@ func (c *ModuleCommand) Help() string {
 Usage: tfupdate module [options] <MODULE_NAME> <PATH>
 
 Arguments
-  MODULE_NAME        A name of module
+  MODULE_NAME        A name of module or a regular expression in RE2 syntax
                      e.g.
                        terraform-aws-modules/vpc/aws
                        git::https://example.com/vpc.git
+                       git::https://example.com/.+
```

Comment:
Since this is an example of regular expression use cases, it is better to escape the dot character correctly. `git::https://example\.com/.+`

Reply:
This is fixed with the commit #9f63ba140c5ccdb8a67f93efc7ba38dbf596454a